app/presenters/history_event.rb

```diff
@@ -0,0 +1,47 @@
+require "delegate"
+
+# HistoryEvent wraps & delegates to C2Version
```

Comment:
I'd say my general philosophy is that the class itself plus related specs should be enough documentation unless something really unusual is going on. If stuff is still confusing, that might mean we need more tests or more clearly written classes. There are def exceptions, though!